CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [Unreleased] - ReleaseDate
+### Added
+- Added `MutexWeak` by using the `sync::Weak`
```

Comment:
The "by using ..." clause isn't easy to understand.  It's also unnecessary.  You can just say "Added `MutexWeak`", or "Added `MutexWeak`, a non-owning reference to a `Mutex`.